documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -120,7 +120,7 @@ docker run -it --rm --name avro-consumer \
       --topic db.myschema.mytable
 ----
 
-[[names]]
+[[avro-naming]]
```

Comment:
The use of `avro.adoc#names` is quite plentiful across upstream docs.  Can you please update the references?  From what I can tell there are references in:

* mongodb.adoc
* mongodb-event-flattening.adoc
* cassandra.adoc
* db2.adoc
* oracle.adoc
* postgresql.adoc (this you already included)
* sqlserver.adoc

Reply:
Ahhh. of course! I'm on it.